.github/actions/release/action.yaml

```diff
@@ -0,0 +1,177 @@
+
+          Checkout the tag:
+          git checkout <latest-existing-patch>
+
+          Create a new branch from the commit:
+          git checkout -b patch/${{inputs.tag-prefix}}/<new-patch>
+
+          Cherry pick current changes to the new branch:
+          git cherry-pick ${{ github.sha }}
+
+          Push the new branch:
+          git push origin <branch-name>
+
+          Create a PR from the new branch to the previous patch tag
+          hub pull-request -b <previous-latest-patch-tag>
+
+          Once the PR is approved, run the release workflow and choose the
+          branch created above as the base branch. Note that this patch will not
+          be part of the main branch unless explicitly merged into it. And none
+          of the commits from the main branch since the last patch will be part
+          of this release.
+        EOF
+          exit 1
+        else
+          echo "Tag is unique. OK to proceed"
+        fi
+
+    - name: Check major and minor releases are from main branch only
+      id: validate-branch
+      shell: bash
+      if:
+        steps.semantic-version.outputs.changed == 'true' &&
+        (steps.semantic-version.outputs.version_type == 'major' ||
+        steps.semantic-version.outputs.version_type == 'minor')
+      run: |
+        if [[ "${{ github.ref }}" != "refs/heads/main" ]]; then
+          echo "Major and Minor releases are only allowed from main branch"
+          exit 1
+        else
+          echo "Release from main branch. OK to proceed"
+        fi
+
+    - name: Edit cargo toml version
+      if: steps.semantic-version.outputs.changed == 'true'
+      shell: bash
+      run: |
+        cargo install toml-cli
+        old_version=$(toml get ./${{ inputs.cargo-toml-path }} package.version)
+        new_verson=${{ steps.semantic-version.outputs.version }}
+        echo "Updating from version $old_version to $new_verson"
+        new_toml=$(toml set ./${{ inputs.cargo-toml-path }} package.version \
+          ${{ steps.semantic-version.outputs.version }})
+        echo "$new_toml" > ${{ inputs.cargo-toml-path }}
+        head -10 ${{ inputs.cargo-toml-path }}
+
+    - name: Create release commit
+      id: create-release-commit
+      shell: bash
+      if:
+        inputs.dry-run == 'false' &&
+        steps.semantic-version.outputs.changed == 'true'
+      run: |
+        version_tag=${{ steps.semantic-version.outputs.version_tag }}
+        release_message="chore(release): $version_tag"
+        git config --local user.email "devops@axelar.network"
+        git config --local user.name "Axelar DevOps"
+        git add .
+        git commit --allow-empty -m "$release_message"
+        git push origin ${{ github.ref }}
+
+    - name: Push Tag
+      id: push-tag
+      shell: bash
+      if:
+        inputs.dry-run == 'false' &&
+        steps.semantic-version.outputs.changed == 'true'
+      run: |
+        echo "Tag: ${{ steps.semantic-version.outputs.version_tag }}"
```

Comment:
Does the `version_tag` here include the name of the repo that we are releasing?

Reply:
sammy you can play around here https://github.com/axelarnetwork/ampd-release-testing

yes it does include the name